src/main/java/com/ibm/cloud/cloudant/kafka/mappers/DocumentToSourceRecord.java

```diff
@@ -28,8 +31,12 @@
 
 public class DocumentToSourceRecord implements BiFunction<String, ChangesResultItem, SourceRecord> {
 
-    // Record key schema is Map<String, String>
-    private static final Schema RECORD_KEY_SCHEMA = SchemaBuilder.map(Schema.STRING_SCHEMA, Schema.STRING_SCHEMA).build();
+    // Record key schema is {"_id": doc_id, "cloudant.url": url, "cloudant.db": db}
+    public static final Schema RECORD_KEY_SCHEMA = SchemaBuilder.struct()
+            .field(CloudantConst.CLOUDANT_DOC_ID, Schema.STRING_SCHEMA)
+            .field(InterfaceConst.DB, Schema.OPTIONAL_STRING_SCHEMA)
+            .field(InterfaceConst.URL, Schema.OPTIONAL_STRING_SCHEMA)
```

Comment:
This is a nice touch, I had been thinking about this also, that making them optional makes it easier to filter them if desired without the need to create a new schema.